gateway/query.go

```diff
@@ -129,6 +133,12 @@ func rewardHandlerFn(rs *RestServer) http.HandlerFunc {
 	}
 }
 
+type RewardRequest struct {
+	MiningReward            *big.Int `json:"mining_reward"`
+	ServiceReward           *big.Int `json:"service_reward"`
+	RewardRequestProtoBytes string   `json:"reward_request_proto_bytes"` // proto msg for reward snapshot from latest intendWithdraw
```

Comment:
add signers, similar to https://github.com/celer-network/sgn/blob/develop/x/validator/client/cli/query.go#L500